tests/pkyaml_test.py

```diff
@@ -36,23 +24,28 @@ def test_load_resource():
     from pykern import pkyaml
 
     p1 = pkunit.import_module_from_data_dir("p1")
-    assert (
-        "v2" == p1.y["f2"]
-    ), "Resource should be loaded relative to root package of caller"
+    pkunit.pkeq(
```

Comment:
This is older code, and there's only one test case and the output is checked inline (not as an output file), since it is so simple. I don't think casedirs could work with multiple tests due to the importlib above. You can only import a module once.